web-client/src/presenter/actions/JudgeActivityReport/getTrialSessionsForJudgeActivityReportAction.ts

```diff
@@ -9,17 +9,16 @@ export const getTrialSessionsForJudgeActivityReportAction = async ({
     state.judgeActivityReport.filters,
   );
 
-  let judgeIdForRequest: string =
-    TEMP_JUDGE_ID_TO_REPRESENT_ALL_JUDGES_SELECTION;
+  let judgeIdForRequest: string = ID_FOR_ALL_JUDGES;
 
   if (judgeName !== 'All Judges') {
-    const allJudges: Array<{
+    const listOfJudges: Array<{
       role?: string;
       userId?: string;
       name?: string;
     }> = get(state.judges);
 
-    const { userId } = (allJudges || []).find(
+    const { userId } = (listOfJudges || []).find(
```

Comment:
Do we need to do || []? Does state.judges not always have at least an empty array? Can we make it start with an empty array?

Reply:
Fixed: [20ee9f9](https://github.com/ustaxcourt/ef-cms/pull/3841/commits/20ee9f9f5539be673cc74815f10f001c23c560dd)